collections/guides/manage_applications_terraform/index.md

```diff
@@ -0,0 +1,160 @@
+    width: 1200
+    height: 630
+---
+
+# Manage Applications with Terraform
+{: .no_toc }
+
+Terraform is a popular and powerful tool that enables you to version your infrastructure as code. In this guide, we'll walk
+you through setting up Terraform to manage your resources locally within Basis Theory. By the end of this guide, you will have
+created an Application under your tenant using Terraform, and you will have an Application key available for you to use.
+
+### Table of contents
+{: .no_toc .text-delta }
+1. 
+{:toc}
+
+<span class="base-alert warning">
+  <span>
+    To start, you'll need a new <code>Management</code> [Application](https://docs.basistheory.com/api-reference/#applications) with <code>application:create</code>, <code>application:read</code>, <code>application:update</code>, and <code>application:delete</code> permissions. <a href="https://portal.basistheory.com/applications/create?name=Terraform+Application&permissions=application%3Acreate&permissions=application%3Aread&permissions=application%3Aupdate&permissions=application%3Adelete&type=management" target="_blank">Click here to create one.</a>
+  </span>
+</span>
+
+## Install Terraform
+
+To achieve the following steps, you must have Terraform CLI installed locally or wherever you're planning on running your configuration.
+The best guide for installing Terraform can be found on their own website [here](https://learn.hashicorp.com/tutorials/terraform/install-cli).
+
+## Set up Terraform configuration
+
+Let's setup the Terraform configuration you'll need for creating an Application and exporting the Application key you'll
+need to interact with other Basis Theory resources.
+
+### Set up the basistheory provider
+
+Firstly, you'll need to configure the `terraform` configuration block in a `main.tf` file. Let's add a source for where
+to pull down our Terraform provider, and pin the version as well.
+
+```terraform
+terraform {
+  required_providers {
+    basistheory = {
+      source  = "basis-theory/basistheory"
+      version = ">= 0.1.3"
+    }
+  }
+}
+```
+
+Now you'll need to define some details for the provider. To start with you can define the API URL, API key, and client timeout
+within the `basistheory` provider.
+
+```terraform
+provider "basistheory" {
+  api_url        = "https://api.basistheory.com"
+  api_key        = "<YOUR_API_KEY_HERE>"
+  client_timeout = 5
+}
+```
+
+Another configuration option is to just set the API key through setting `BASISTHEORY_API_KEY` environment var. This leaves
+you with optionally setting the API URL and client timeout through `BASISTHEORY_API_URL` and `BASISTHEORY_CLIENT_TIMEOUT`
+env vars, respectively. These don't need to be explicitly set since the API URL defaults to `https://api.basistheory.com`
+and the client timeout defaults to 15 seconds. So the most minimal `provider` config can look like this:
+
+```terraform
+provider "basistheory" {
+}
+```
```

Comment:
`terraform {}` and `provider {}` blocks are almost always configured in `main.tf`. Should we combine both blocks in the minimal config example and rename to `main.tf`?